incident_io/manifest.json

```diff
@@ -0,0 +1,67 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "95ee2e88-d2ee-45f8-a4d6-2cb9eced79ee",
+  "app_id": "incident-io",
+  "display_on_public_website": false,
```

Comment:
Note that this means there's no page for this integration in the public docs. Is this something that's ok?

Reply:
Understood!